routing/routes.go

```diff
@@ -649,9 +649,10 @@ func NewRoutes() (r *Routes) {
 			handlers.WorkloadDashboard,
 			true,
 		},
-		// swagger:route GET /namespaces/{namespace}/apps/{app}/customdashboard/{template} apps customDashboard
+		// swagger:route GET /namespaces/{namespace}/customdashboard/{dashboard} customDashboard
```

Comment:
I see we move from

```
/namespaces/{namespace}/apps/{app}/customdashboard/{template}
```

to

```
/namespaces/{namespace}/customdashboard/{dashboard}
```

Why did it need the {app} before and now only {dashboard} ?

Perhaps it's documented in other place, but I couldn't see it, and without being familiar with the implementation that came to my attention.

Reply:
Ok, I read below it has been moved as query parameters.

Reply:
That's right, I forgot to document. Just pushed a commit about that : https://github.com/kiali/kiali/pull/1033/commits/9f28eae5173176408a5b8d44fff0d424ea8dc9fb

Basically, what happens is that k-charted has a more generic use of labels than we used to have, and now `app` is just no more than a part of generic label filtering. These filters are now provided as query params.

So, to reuse the helper endpoint built in k-charted, it has to be simplified in our route as well.